mitmproxy/tools/cmdline.py

```diff
@@ -568,23 +568,23 @@ def proxy_ssl_options(parser):
 def onboarding_app(parser):
     group = parser.add_argument_group("Onboarding App")
     group.add_argument(
-        "--noapp",
-        action="store_false", dest="app",
+        "--noonboarding",
```

Comment:
I'd prefer `--no-onboarding` - IMHO this is a more common way to negate options.